src/TestInfrastructure/Framework/ConfigurableVsProjectSystemHelper.cs

```diff
@@ -121,6 +121,17 @@ public void ClearProjectProperty(Project dteProject, string propertyName)
             projMock.ClearBuildProperty(propertyName);
         }
 
+        public IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
+        {
+            ProjectMock dteProject = hierarchy as ProjectMock;
+            if(dteProject == null)
+            {
+                Assert.Inconclusive($"Only expecting {nameof(ProjectMock)} type");
```

Comment:
The eternal Inconclusive/Fail debate...
Eyal advised me to use inconclusive [here](https://github.com/SonarSource-VisualStudio/sonarlint-visualstudio/pull/68#discussion-diff-61053993) for the same thing previously.

I think this depends of what test 'inconclusive/failure' means. Is it that the _test code_ is broken, or is it that the _product_ failing? In the case of the later then `Assert.Inconclusive` makes sense since there's a problem in the test data/setup, rather than the product.